config/bidderinfo.go

```diff
@@ -320,8 +320,13 @@ func processBidderAliases(aliasNillableFieldsByBidder map[string]aliasNillableFi
 		if aliasBidderInfo.PlatformID == "" {
 			aliasBidderInfo.PlatformID = parentBidderInfo.PlatformID
 		}
-		if aliasBidderInfo.Syncer == nil {
-			aliasBidderInfo.Syncer = parentBidderInfo.Syncer
+		if aliasBidderInfo.Syncer == nil && parentBidderInfo.Syncer != nil {
```

Comment:
I have a question regarding a case you laid out in your PR description: 

```
Case 2: parent has not defined syncer key
Expected: Alias should inherit entire Parent syncer cfg with parent name as syncer key for alias.
```
Would this be referring to scenario where `parentBidderInfo.Syncer == nil`? And if so, where in this PR is the code that allows the alias to inherit the entire parent syncer cfg?

Reply:
@AlexBVolcy, we leverage on existing `BuildSyncer()` function to inherit parent cfg for alias.

As per adapter aliases feature, bidders (parent. + aliases ) can share user sync cfg as follow:
- parent should define user sync config
- alias should define parent name or parent syncer key in syncer cfg

We have `BuildSyncer()` function that,

1. validates above adapter aliases rule
2. groups all bidders with same key together (parent and aliases are grouped together)
3. assigns  parent syncer config to all bidders grouped on step 2

https://github.com/prebid/prebid-server/blob/831dd711a23f1c32b7829f0843abee45126fc312/usersync/syncersbuilder.go#L28

In this way,  alias inherit the entire parent syncer cfg.  Case where  `parentBidderInfo.Syncer == nil` is already handled by  `BuildSyncer()`. So changes were needed as part of this PR.